lib/models/device.coffee

```diff
@@ -1052,40 +1045,36 @@ getDeviceModel = (deps, opts) ->
 	# @param {String} applicationName - application name
 	# @param {String} uuid - device uuid
 	#
-	# @fulfil {Object} - device
+	# @fulfil {Object} - device registration info
 	# @returns {Promise}
 	#
 	# @example
-	# resin.models.device.generateUUID().then(function(uuid) {
-	# 	resin.models.device.register('MyApp', uuid).then(function(device) {
-	# 		console.log(device);
-	# 	});
+	# var uuid = resin.models.device.generateUniqueKey();
+	# resin.models.device.register('MyApp', uuid).then(function(registrationInfo) {
+	# 	console.log(registrationInfo);
 	# });
 	#
 	# @example
-	# resin.models.device.generateUUID(function(error, uuid) {
+	# var uuid = resin.models.device.generateUniqueKey();
+	# resin.models.device.register('MyApp', uuid, function(error, registrationInfo) {
 	# 	if (error) throw error;
-	#
-	# 	resin.models.device.register('MyApp', uuid, function(error, device) {
-	# 		if (error) throw error;
-	#
-	# 		console.log(device);
-	# 	});
+	# 	console.log(registrationInfo);
 	# });
 	###
-	exports.register = onlyIf(not isBrowser or isTest) (applicationName, uuid, callback) ->
```

Comment:
As far as I remember I've introduced `isTest` only to workaround this specific limitation. If true good to remove it from everywhere